pkg/mq/msgstream/mqwrapper/kafka/kafka_consumer.go

```diff
@@ -149,7 +149,11 @@ func (kc *Consumer) Chan() <-chan mqwrapper.Message {
 							kc.skipMsg = false
 							continue
 						}
-						kc.msgChannel <- &kafkaMessage{msg: e}
+
+						select {
+						case kc.msgChannel <- &kafkaMessage{msg: e}:
+						case <-kc.closeCh:
```

Comment:
We'd better send a signal to closeCh to close the consumer connection, or it might lead to a resource leak.